src/app/modules/workers/components/workers-filters/workers-filters.component.ts

```diff
@@ -15,11 +16,19 @@ export class WorkersFiltersComponent implements OnInit {
   public form = new FormGroup({
     name: new FormControl(''),
     age: new FormControl(null),
+    city: new FormControl(''),
   });
 
-  ngOnInit(): void {
-    this.form.valueChanges.subscribe(value => {
+  ngOnInit() {
+    this.form.valueChanges.subscribe((value: EmployeeFiltersState) => {
       this.changed.emit(value);
     });
+
+    const { name, age, city } = this.state;
+    this.form.setValue({
```

Comment:
Nie ma tu pełnej synchronizacji stanu i formularza, ten kod wywoła się tylko raz, a gdyby jakiś inny komponent zmienił query params to ten komponent by na to nie zaregował. Najlepiej obsłużyć to przez setter i getter albo poprzez ngOnChanges